Nextcloud mac ios update (#1) (#434)
Start9Labs/documentation: site/source/user-manual/service-guides/nextcloud/nextcloud-setup/nextcloud-ios.rst

=========
Make sure you have first set up :ref:`LAN access<lan-ios>`.

> 1. Download the appropriate iOS client from `App Store <https://apps.apple.com/us/app/nextcloud/id1125420102>`_.

Reviewer: Lines 15-18 can probably be deleted, as they are irrelevant here.  This is probably also the case for Android.

Author: Lines deleted

2. Open the client and click "Log In".
   

>    .. figure:: /_static/images/services/nextcloud/nextcloud-iOS-step2.png

Reviewer: This image is broken because the file name is .PNG.PNG (simply rename the file to fix)

    :width: 40%
    :alt: nextcloud-ios-login   
  
3. From your server's Nextcloud Service page, go to "Interfaces" and copy the LAN address.
   

>    .. figure:: /_static/images/services/nextcloud/nextcloud-stabdalone-client-macos-step3.png

Reviewer: These images are all named 'stabdalone' instead of 'standalone'

The images from step 3-8 are all broken, possibly because you use .PNG and not .png

    :width: 40%
    :alt: nextcloud-ios-login

> 4. Enter your LAN address under "Server Address" and click "Next".

Reviewer: ```suggestion
4. Paste your LAN address under "Server Address" and click "Next".
```

   
   .. figure:: /_static/images/services/nextcloud/nextcloud-iOS-step3.png
    :width: 40%
    :alt: nextcloud-ios-login

> 5. You will be shown the "Connect to your account" screen, and then click "Log In".

Reviewer: ```suggestion
5. You will be shown the "Connect to your account" screen.  Tap "Log In".
```

   .. figure:: /_static/images/services/nextcloud/nextcloud-iOS-step4.png
    :width: 40%
    :alt: nextcloud-ios-login

> 6. Next, you will be asked to enter the username and password, which can be found on the startOS Nextcloud Service page under the "Properties" tab.

Reviewer: ```suggestion
6. Next, you will be asked to enter the username and password.  You can find these on your StartOS Nextcloud service page on the "Properties" tab.
```

   .. figure:: /_static/images/services/nextcloud/nextcloud-iOS-step5.png
    :width: 40%
    :alt: nextcloud-ios-login

> 7. The account access page will be displayed, then click "Grant access".

Reviewer: ```suggestion
7. The account access page will be displayed, and you can simply tap "Grant access".
```

   .. figure:: /_static/images/services/nextcloud/nextcloud-iOS-step6.png
    :width: 40%
    :alt: nextcloud-ios-login

8. Files will begin syncing immediately, and you will see your username and account icon in the top left corner.

   .. figure:: /_static/images/services/nextcloud/nextcloud-iOS-step7.png
    :width: 40%
    :alt: nextcloud-ios-login
    

> 9.  That's it! From this desktop client you will recieve notifications, control accounts and syncing, and quickly access your Apps' WebUI pages

Reviewer: ```suggestion
9.  That's it! From this mobile client you can receive notifications, control accounts and syncing, and quickly access the web UIs for your Nextcloud apps.
```

Device Integration
------------------

> In order to sync calendars and contacts with your iOS device, follow the steps below, which are adapted from the `Official Nextcloud guide <https://docs.nextcloud.com/server/25/user_manual/en/groupware/sync_ios.html>`_.  First head into the top-righthand menu of your Nextcloud's WebUI and click "Apps," then search for and install the Calendar and/or Contacts Apps. Next folow this guide for iOS.

Reviewer: I believe the above 'Tor Setup' section is wrong.  I had copy-pasted the desktop setup to android/ios and I have already fixed the android section.  It references the desktop and I think the directions are wrong (should be much simpler with Orbot) - can you verify and fix please?

Reviewer: It sounds like there may be general issues with iOS/Orbot connections right now.  One user suggested that uninstall/re-install of Orbot fixed for him.

1. Open the "Settings" app.

  .. note:: You will need to perform 2 individual setups, one for Calendar and one for Contacts.

> 2. Select "Calendar" (or "Contacts") -> "Accounts" -> "Add Account" -> "Other" -> "either CalDAV (for Calendar setup) or CardDAV (for Contacts setup)".Return to this step after completing one in order to add the other.

Reviewer: ```suggestion
2. Select "Calendar" (or "Contacts") -> "Accounts" -> "Add Account" -> "Other" -> "either CalDAV (for Calendar setup) or CardDAV (for Contacts setup)".  Return to this step after completing one in order to add the other.
```

  .. figure:: /_static/images/services/nextcloud/native-nextcloud-iOS-step1.PNG
    :width: 50%
    :alt: nextcloud account settings

3. Enter the following fields and clik "Next":

>   .. figure:: /_static/images/services/nextcloud/native-nextcloud-iOS-step2.PNG

Reviewer: This image is also broken
